src/daemon/database/schema.rs

```diff
@@ -68,20 +69,47 @@ CREATE TABLE presigned_transactions (
     vault_id INTEGER NOT NULL,
     type INTEGER NOT NULL,
     psbt BLOB UNIQUE NOT NULL,
+    txid BLOB UNIQUE NOT NULL,
     fullysigned BOOLEAN NOT NULL CHECK (fullysigned IN (0,1)),
     FOREIGN KEY (vault_id) REFERENCES vaults (id)
         ON UPDATE RESTRICT
         ON DELETE RESTRICT
 );
 
+/* A bridge between the Unvault transactions a Spend transaction
+ * may refer and the possible Spend transactions an Unvault one
+ * may be associated with.
+ */
+CREATE TABLE spend_inputs (
+    id INTEGER PRIMARY KEY NOT NULL,
+    unvault_id INTEGER NOT NULL,
+    spend_id INTEGER NOT NULL,
+    FOREIGN KEY (unvault_id) REFERENCES presigned_transactions (id)
+        ON UPDATE RESTRICT
+        ON DELETE RESTRICT,
+    FOREIGN KEY (spend_id) REFERENCES spend_transactions (id)
+        ON UPDATE RESTRICT
+        ON DELETE RESTRICT
+);
+
+/* This stores Spend transactions we created. A txid row is there to
+ * ease research.
+ */
+CREATE TABLE spend_transactions (
+    id INTEGER PRIMARY KEY NOT NULL,
+    psbt BLOB UNIQUE NOT NULL,
+    txid BLOB UNIQUE NOT NULL,
+    broadcast BOOLEAN NOT NULL CHECK (broadcast IN (0,1))
```

Comment:
Maybe move to a ternary status: 0, 1, 2 (broadcasted)

Reply:
Done